modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -1508,5 +1594,33 @@ Status description:: error: procedure exception - procedure sandboxed. `{ $sig }
 
 Status description:: error: procedure exception - procedure compilation failed. Failed to compile procedure/function defined in `{ $procClass }`: `{ $msg }`
 
+=== 52N36
+
+Status description:: error: procedure exception - invalid procedure argument with API documentation hint. Invalid argument `{ $value }` for `{ $procParam }` on procedure `{ $proc }`. The expected format of `{ $procParam }` is outlined in the `{ $procParamFmt }` API documentation.
```

Comment:
```suggestion
Status description:: error: procedure exception - invalid procedure argument with API documentation hint. Invalid argument `{ $field }` for `{ $procParam }` on procedure `{ $proc }`. The expected format of `{ $procParam }` is outlined in the `{ $procParamFmt }` API documentation.
```

This one was different in spreadsheet and implementation, I have updated the spreadsheet.